src/get_games.py

```diff
@@ -144,5 +145,16 @@ def download_games(name: str,
 
 
 if __name__ == '__main__':  # pragma: no cover
+    auth = setup(path="./")
+
+    user_stats = lichess.api.user('carequinha')
```

Comment:
Maybe add a variable user, so you can change it more easily (see also line 157).

Reply:
Fixed.